backend.native/tests/build.gradle

```diff
@@ -3335,6 +3335,12 @@ Task interopTest(String name, Closure<KonanInteropTest> configureClosure) {
     }
 }
 
+
+standaloneTest("interop_objc_allocException") {
```

Comment:
Should be marked as depending on platform libs. See e.g. https://github.com/JetBrains/kotlin-native/blob/ec6b4d6d92d512c3b70786d75fc6ba97d3667cb7/backend.native/tests/build.gradle#L3390